cmd/minishift/cmd/config/view_test.go

```diff
@@ -74,6 +77,10 @@ func TestConfigView(t *testing.T) {
 
 		template := determineTemplate(tt.template)
 		configView(configTest, template, tee.StdoutBuffer)
-		assert.Equal(t, tt.expectedString, tee.StdoutBuffer.String())
+		var b bytes.Buffer
+		writer := bufio.NewWriter(&b)
+		formatAndWriteLines(writer, strings.Split(tt.expectedString, "\n"))
```

Comment:
In this case you would also have to test `formatAndWriteLines` in exclusion of the rest of code...

Reply:
Cool, I'll write a test case for this function